examples/tutorials/Generating_molecules_with_MolGAN.ipynb

```diff
@@ -10,7 +10,7 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.                invalid_mask = (flattened_adjacency &lt; 0) | (flattened_adjacency &gt;= gan.edges)
</pre><p>Add a comment above this adding an explanation</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deepchem/deepchem/pull/3994/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>